src/Reader.php

```diff
@@ -69,6 +70,7 @@ public static function readFromYaml(string $yaml, string $baseType = OpenApi::cl
      * @throws TypeErrorException in case invalid spec data is supplied.
      * @throws UnresolvableReferenceException in case references could not be resolved.
      * @throws IOException when the file is not readable.
+     * @throws InvalidJsonPointerSyntaxException in case an invalid JSON pointer string is passed to the spec references
```

Comment:
Small nitpick: add a dot at the end of the line, for consistency with the preceding lines.

Reply:
Done ;)